Makefile

```
@@ -90,3 +90,6 @@ dist: clean ## builds source and wheel package
 
 install: clean ## install the package to the active Python's site-packages
 	python3 setup.py install
+
+constraints.txt: requirements.txt
```

Since you're not duplicating the setup.py requirements in requirements.txt, this is also required:

```suggestion
constraints.txt: requirements.txt setup.py
```